.github/workflows/test.yml

```diff
@@ -68,21 +68,35 @@ jobs:
         - { os: windows-2019, ruby: '1.9' }
         - { os: windows-2022, ruby: '1.9' }
         - { os: windows-2025, ruby: '1.9' }
-        - { os: windows-2019, ruby: debug }
-        - { os: windows-2022, ruby: debug }
-        - { os: windows-2025, ruby: debug }
+        - { os: windows-11-arm, ruby: '1.9' }
+        - { os: windows-11-arm, ruby: '2.0' }
+        - { os: windows-11-arm, ruby: '2.1' }
+        - { os: windows-11-arm, ruby: '2.2' }
+        - { os: windows-11-arm, ruby: '2.3' }
+        - { os: windows-11-arm, ruby: '2.4' }
+        - { os: windows-11-arm, ruby: '2.5' }
+        - { os: windows-11-arm, ruby: '2.6' }
+        - { os: windows-11-arm, ruby: '2.7' }
+        - { os: windows-11-arm, ruby: '3.0' }
+        - { os: windows-11-arm, ruby: '3.1' }
+        - { os: windows-11-arm, ruby: '3.2' }
+        - { os: windows-11-arm, ruby: '3.3' }
```

Comment:
These should be in their own category like `# Not available on Windows ARM` as they are a bit different than the other exclusions which are "not available on Windows any arch"